Class/A-Objects/exercise2.js

```diff
@@ -9,9 +9,14 @@ let dog1 = {
 
 // Define your own Object called dog2 with the same properties as the object above
 // The properties should have different values
-
+let dog2 = {
+    breed: "Big",
+    name: "Pochi",
+    isHungry:false,
+    happiness:9
+}
 
 // Log the value of each property
-
+console.log(dog2.breed,dog2.name,dog2.isHungry,dog2.happiness);

```

Comment:
Hi Megumi, this is all right,
just for the future, to make your life easier, you might want to console.log() in more explanatory way, something like:
```javascript
console.log(`name: ${dog2.name}, breed: ${dog2.breed}, hungry? ${dog2.isHungry? "Yes" : "No"}, happiness: ${dog2.happiness}`);
```